packages/sdks/web-js-sdk/manualLicense.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Let's remove this, since it is not MIT

Reply:
I just checked that it's working
Removed